src/main/java/mekanism/common/item/ItemAtomicDisassembler.java

```diff
@@ -365,8 +368,7 @@ public void loop(Coord4D pointer) {
                 Coord4D coord = pointer.offset(side);
                 ItemStack blockStack = coord.getBlock(world).getPickBlock(coord.getBlockState(world), rayTraceResult, world, coord.getPos(), player);
                 if (coord.exists(world) && checkID(coord.getBlock(world)) &&
-                    (stack.isItemEqual(blockStack) || (coord.getBlock(world) == startBlock && OreDictCache.getOreDictName(stack).contains("logWood")
-                                                       && coord.getBlockMeta(world) % 4 == stack.getItemDamage() % 4))) {
+                    (ItemHandlerHelper.canItemStacksStack(stack, blockStack) || (coord.getBlock(world) == startBlock && isWood && coord.getBlockMeta(world) % 4 == stack.getItemDamage() % 4))) {
```

Comment:
that `meta % 4` check should probably get a `TODO` for visiting later - as it would only be relevant to vanilla logs (and I'm not 100% sure on what it does anyway, something to do with the rotations perhaps)